### PR TITLE
Organisation and ReadMe

### PR DESCRIPTION
**Pull request by XhorxhinaT**
Updated!!!!.

## Description

The structure is divided under bin into sut, workload and analyzer. 
ReadMe explains how to run the tool. #79 fulfilled
## Related Issue

The changes done after Sunday need to be as well adapted and updated.

## Checklist

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Path fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,115 @@
 # Cloud Prototyping 2019/2020
 
-## Motivation
 
-This repository was created to automate the benchmarking process of ethereum with caliper. We have created the following docker images and automation scripts to make the process portable and repeatable as possible.
 
-## Docker
```
Comment (eltalkarim) [inline README.md:7]: It is fine to leave it but maybe in a later section

File: README.md [hunk 2]
```diff
@@ -1,43 +1,115 @@
...
+#A benchmarking-based approach to maximize throughput of the private Ethereum
 
-we import from [ethereum/client-go:alltools-stable](https://hub.docker.com/r/ethereum/client-go)  as it has most of the tools we need.
 
-So far we have three images:
+Our goals were to achieve :
+1) Performance Benchmarking. Benchmarking the performance of private Ethereum with Proof-of-Authority consensus  while maximizing Throughput depending on Block size and Block interval.
```
Comment (eltalkarim) [inline README.md]: Some words have double spacing. Throughput should not be capital as it does not start in the beginning of the sentence.

File: README.md [hunk 3]
```diff
@@ -1,43 +1,115 @@
...
-### Ethereum Node Image
-Takes for input the genesis file and bootnode (just paste address in the bootnode file) and start a node on the default port. You don't need to create an account for this node as it fully autoamted withing the image.
 
-### Sealer Node Image
-Takes for input the genesis file and bootnode (just paste address in the bootnode file) and start a mining node that could seal on the default port. It also generated the genesis file accordingly. As Sealers should be in the genesis.json.
+##Getting started
+
+These instruction will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+git clone https://github.com/Cloud-Prototyping-WS-2019-20/cp_ws_1920.git
```
Comment (eltalkarim) [inline README.md:71]: Git clone goes without saying. No need to mention that.

File: README.md [hunk 4]
```diff
@@ -1,43 +1,115 @@
...
+
+
+##Prerequisites
+
+The script checks if the needed dependencies are installed or how to install them. It also runs the script  create-template.sh  to create VM templates.
+
+Run: sh prerequisites.sh
+
+
+Dependency
```
Comment (eltalkarim) [inline README.md]: please mention which version of python.

**Review by eltalkarim — CHANGES_REQUESTED** [review]
No need to push the results. Please add *.zip to the git ignore

File: README.md (hunk 1 above)
Comment (XhorxhinaT) [reply README.md:7]: But we decided to remove that folder of docker

**Review by fayanpardo — DISMISSED** [review]
I do not know how you uploaded this branch but it had lots and lots and lots of duplicated code and version mismatches. Not working at all the execution. I fixed them because we can not delay the rest of branches. It would be ideal if everyone checks I did not leave behind any update and to check if it runs.

**Review by fayanpardo — DISMISSED** [review]

**Comment by XhorxhinaT** [comment]
> I do not know how you uploaded this branch but it had lots and lots and lots of duplicated code and version mismatches. Not working at all the execution. I fixed them because we can not delay the rest of branches. It would be ideal if everyone checks I did not leave behind any update and to check if it runs.

I know, it had, i do not know why wasn't taking only the files i needed. anyway thank u for fixing it even if you did not  have to do that. the only important thing was readMe, sut, workload, analyzer the rest was just from master

File: README.md (hunk 1 above)
Comment (eltalkarim) [reply README.md:7]: we can keep it is okay but just say it is if we want to run in docker for future work

**Review by eltalkarim — APPROVED** [review]

**Pull request merged**